### DISCUSSION TITLE
<Use a descriptive title here>

### DISCUSSION THREAD

**Discussion by HPL001** [idaholab/moose#29378] [Q&A Getting Started]
### Check these boxes if you have followed the posting rules.

- [X] I have consulted the [Posting Guidelines](https://github.com/idaholab/moose/discussions/18270).
- [X] I have searched the [Discussions Forum](https://github.com/idaholab/moose/discussions) and [MOOSE Framework Troubleshooting](https://mooseframework.inl.gov/help/troubleshooting.html) and have not found what I was looking for
- [X] Q&A Getting Started is the most appropriate category for my question (trouble installing, beginner user, ...)

### Issue or question about MOOSE

Dear developers, I installed moose on wsl and then VScode on Windows. I started the code on wsl, and installed the moose related support plug-in, but I was prompted the following problems, I do not know how to solve. For example, I don't have this file, so I can't go to the definition in VScode.
![image](https://github.com/user-attachments/assets/8a39b36e-e034-4ca1-94e1-48f02d14b9be)

**Comment by GiudGiud** [comment]
Hello

You can perform these commands with any moose application executable, preferably your application executable. 
Then place the output in the folder you opened VSCode with

Guillaume

**Reply by HPL001** [reply]
Thank you very much for your help, I solved this problem under your guidance.
